### ISSUE TITLE
possible errors in cellProp?

### ISSUE THREAD

**Issue by jspaaks** [NLeSC/enram#80]
I'm getting this sample output: ``iRangOfMax`` and ``iAzimOfMax`` should not be ``-1``. Also ``.clutterArea/.area`` is pretty big for the first cell (index=2). Maybe related to #37?
```
Found 9 cells.
#Cell analysis for elevation 0.300000:
#Minimum cell area in pixels   : 4
#Threshold for mean dBZ cell   : 15 dBZ
#Threshold for mean stdev cell : 5 dBZ
#Valid cells                   : 5/9
#
cellProp: .index .area .clutterArea .dbzAvg .texAvg   .cv .dbzMax .iRangOfMax .iAzimOfMax .drop
cellProp:      2   118          101   39.29    1.94 0.05   61.00           3          12     F
cellProp:      3    17           17    0.00    0.00 0.00  -31.50          -1          -1     F
cellProp:      4    12           12    0.00    0.00 0.00  -31.50          -1          -1     F
cellProp:      5    11           11    0.00    0.00 0.00  -31.50          -1          -1     F
cellProp:      6     8            8    0.00    0.00 0.00  -31.50          -1          -1     F
```

**Comment by adokter** [comment]
Reopen issue #39:

condition 1: if `cellProp[iCell].area < areaMin` then drop the cell
condition 2: 
```
if (cellProp[iCell].dbzAvg < cellDbzMin && cellProp[iCell].texAvg > cellStdDevMax
   if cellProp[iCell].clutterArea / cellProp[iCell].area) < cellClutterFraction
        drop the cell
   else
        do not drop the cell
   endif
endif
```

**Comment by jspaaks** [comment]
Seems to have been due to rows and columns being interchanged when mapping the data from the RAVE object to libvol2bird. Solved by commit be5ff22024e0c4a78df7fe813ea2bd62952182f7. Closing
